agent/chaldea/__init__.py

```
@@ -32,30 +34,42 @@ def fetch_share_data(source: str):
 
     if direct_data:
         logger.info("[Chaldea] 匹配到长链接数据特征，开启离线解码...")
+        mfaalog.info("[Chaldea] 离线解码 direct_data...")
         share_data = decode_content(direct_data)
         team_id = "offline"
         quest_id = (share_data.get("quest") or {}).get("id", "0") if share_data else "0"
     elif team_id:
+        mfaalog.info(f"[Chaldea] 按 team_id={team_id} 下载队伍...")
         team_resp = fetch_team_by_id(team_id)
         if team_resp and "content" in team_resp:
+            mfaalog.info(f"[Chaldea] 队伍下载成功, 开始解码 content")
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)